pyproject.toml

```diff
@@ -47,6 +47,16 @@ readthedocs-sphinx-search = "^0.1.0"
 toml = "^0.10.2"
 sphinx-automodapi = "^0.13"
 invoke = "^1.6.0"
+codecov = "^2.1.11"
+rmtest = "^0.7.0"
+nose = "^1.3.7"
```

Comment:
Why all the python dependencies we shouldn't be using?  Specificaally, ml2rt, scikit-image, and numpy I expect as dependencies (since they're needed for ai>

But, the others we shouldn't need, as this wraps into our test harness. Let's specifically eliminate rmtest, nose, codecov, and six.